routers/api/v1/api.go

```diff
@@ -897,7 +897,7 @@ func Routes() *web.Route {
 						m.Get(".patch", repo.DownloadPullPatch)
 						m.Post("/update", reqToken(), repo.UpdatePullRequest)
 						m.Combo("/merge").Get(repo.IsPullRequestMerged).
-							Post(reqToken(), mustNotBeArchived, bind(auth.MergePullRequestForm{}), repo.MergePullRequest)
```

Comment:
struct should be moved into /modules/structs - if it is needed for webUI too copied

Reply:
I don't think move all structs to structs module is a good idea. In fact, if there is any other chance, I think move structs to other modules is better. But I have no idea now.